samples/openthread/coap_client/README.rst

```diff
@@ -113,22 +113,17 @@ Configuration
 
 .. _coap_client_sample_activating_variants:
 
-Configuration files
-===================
+Snippets
+========
 
-The sample provides predefined configuration files for typical use cases, and to activate sample extensions.
-You can find the configuration files in the root directory of the sample.
+.. include:: /includes/sample_snippets.txt
 
-Specify the corresponding file names in the :makevar:`OVERLAY_CONFIG` option when building.
-See :ref:`cmake_options` for instructions on how to add this option.
-For more information about using configuration overlay files, see :ref:`zephyr:important-build-vars` in the Zephyr documentation.
+The following snippets are available:
```

Comment:
It was not listed before, but `debug` snippet is also available.

Reply:
right! thanks for catching. Added a line here and in other samples